src/types/fastlane.ts

```diff
@@ -97,4 +97,55 @@ export interface IFastlaneInstance {
     };
     FastlanePaymentComponent: (options: IFastlaneComponentOptions) => Promise<IFastlanePaymentComponent>;
     FastlaneCardComponent: (options: Omit<IFastlaneComponentOptions, 'shippingAddress'>) => IFastlaneCardComponent;
+}
+
+export interface IFastlaneOptions {
+    shippingAddressOptions: IFastlaneAddressOptions,
+    cardOptions: IFastlaneCardOptions,
+    styles: IFastlaneStyleOptions
+}
+
+export interface IFastlaneAddressOptions {
+    // default: empty array = all locations allowed
+    allowedLocations: [string];
+}
+
+export interface IFastlaneCardOptions {
+    // default: empty array = all brands allowed
+    allowedBrands: [IFastlaneCardBrandEnum];
```

Comment:
This type syntax describes a field that takes an array with only one element that is of "type" `IFastlaneCardBrandEnum`. Is this correct or is this field an array of "type" `IFastlaneCardBrandEnum`?

```suggestion
    allowedBrands: IFastlaneCardBrandEnum[];
```

Reply:
I was just copying Paypal's docs, but you're right, it should be `IFastlaneCardBrandEnum[]`. Updated.